docs/index.rst

```diff
@@ -1,25 +1,18 @@
-.. python-gitlab documentation master file, created by
-   sphinx-quickstart on Mon Dec  8 15:17:39 2014.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
-
-Welcome to python-gitlab's documentation!
-=========================================
-
-Contents:
+.. include:: ../README.rst
```

Comment:
This tries to deduplicate the intro a bit between our readme and our docs while making it look like a real website. I don't really like generating the whole TOC tree as a page if we already have it in the sidebar :thinking: